src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -509,23 +517,37 @@ std::string statisticsTreat
     out += TAG_ADD_STRING("subCache",          listBuffer);
   }
 
+  std::string timingStatString      = timingStatistics(indent2, ciP->outFormat, ciP->apiVersion);
+  bool        timingStat            = (timingStatString != "");
+  bool        threadpool            = strcmp(notificationMode, "threadpool") == 0;
+  bool        commaAfterThreadpool  = timingStat;
+  bool        commaAfterSubCache    = timingStat || threadpool;
+
   out += TAG_ADD_INTEGER("subCacheRefreshs", mscRefreshs, true);
   out += TAG_ADD_INTEGER("subCacheInserts",  mscInserts,  true);
   out += TAG_ADD_INTEGER("subCacheRemoves",  mscRemoves,  true);
   out += TAG_ADD_INTEGER("subCacheUpdates",  mscUpdates,  true);
-  out += TAG_ADD_INTEGER("subCacheItems",    cacheItems,  false);
+  out += TAG_ADD_INTEGER("subCacheItems",    cacheItems,  commaAfterSubCache);
 
   if (strcmp(notificationMode, "threadpool") == 0)
   {
-    out += TAG_ADD_INTEGER("noOfNotificationsQueueIn", QueueStatistics::getIn(), true);
-    out += TAG_ADD_INTEGER("noOfNotificationsQueueOut", QueueStatistics::getOut(), true);
-    out += TAG_ADD_INTEGER("noOfNotificationsQueueReject", QueueStatistics::getReject(), true);
-    out += TAG_ADD_INTEGER("noOfNotificationsQueueSentOK", QueueStatistics::getSentOK(), true);
-    out += TAG_ADD_INTEGER("noOfNotificationsQueueSentError", QueueStatistics::getSentError(), false);
+    out += TAG_ADD_INTEGER("noOfNotificationsQueueIn",        QueueStatistics::getIn(),        true);
+    out += TAG_ADD_INTEGER("noOfNotificationsQueueOut",       QueueStatistics::getOut(),       true);
+    out += TAG_ADD_INTEGER("noOfNotificationsQueueReject",    QueueStatistics::getReject(),    true);
+    out += TAG_ADD_INTEGER("noOfNotificationsQueueSentOK",    QueueStatistics::getSentOK(),    true);
+    out += TAG_ADD_INTEGER("noOfNotificationsQueueSentError", QueueStatistics::getSentError(), commaAfterThreadpool);
+  }
+
+
+  //
+  // Timing statistics ONLY if JSON
+  //
+  if (ciP->outFormat == JSON)
```

Comment:
Yeah, I decided not to waste time on XML for this new feature.
If/When we rewrite the statistics, I also believe it's a good idea to do it ONLY for JSON.